acquire/acquire.py

```diff
@@ -1264,6 +1264,13 @@ class RemoteAccess(Module):
         # anydesk - Mac + Linux
         ("glob", ".anydesk*/*", from_user_home),
         ("file", "/var/log/anydesk.trace"),
+        # RustDesk - Windows
+        ("dir", "sysvol/ProgramData/RustDesk"),
+        ("dir", "AppData/Roaming/RustDesk/log/server/", from_user_home),
+        # RustDesk - Mac + Linux
+        ("dir", ".local/share/logs/RustDesk/server/*", from_user_home),
```

Comment:
Was this the intention?
```suggestion
        ("glob", ".local/share/logs/RustDesk/server/*", from_user_home),
```